autoload/youcompleteme.vim

```diff
@@ -67,6 +67,7 @@ function! youcompleteme#Enable()
   endif
 
   call youcompleteme#EnableCursorMovedAutocommands()
+  call youcompleteme#EnableTextChangedAutocommands()
```

Comment:
I mean that if you add a new pair of Enable/Disable functions for the TextChangedI event and I'm using a plugin which called the `EnableCursorMovedAutocommands` and `DisableCursorMovedAutocommands` for performance reason, now if I'm on a version of vim which have the `TextChangedI` event that plugin will need to be updated.